postcode_lookup/templates/index.html

```diff
@@ -64,13 +64,22 @@ <h1 class="c-hero__title">
                             return urlParams.get(param);
                         }
 
-                        const errorParam = getQueryParam('invalid-postcode');
-                        if (errorParam === '1') {
-                            // Display the error message
-                            document.getElementById('dc-postcode-search-form').classList.add("error");
-                            postcodeInput.setAttribute("aria-invalid", "true")
+                        // Function to handle error message display
+                        function handleErrorDisplay(param, message = null) {
+                            if (getQueryParam(param) === '1') {
+                                // display error message
+                                document.getElementById('dc-postcode-search-form').classList.add("error");
+                                postcodeInput.setAttribute("aria-invalid", "true");
+                                // optionally customize message text
+                                if (message) {
+                                    document.getElementById('postcode-search-error-message').innerHTML = message;
+                                }
+                            }
                         }
 
+                        handleErrorDisplay('invalid-postcode');
+                        handleErrorDisplay('api-error', "Sorry, there's a problem with this service at the moment. We're working to fix it as soon as possible.");
```

Comment:
We don't have an error for `InvalidUPRNException` here (which sends you back here with `invalid-uprn=1`), but I kind of don't hate what we do here, which is just dump the user back to the search page. I think that is OK enough that we just leave it for now.